src/lib/TranscriptEditor/MediaPlayer/index.js

```diff
@@ -443,18 +400,17 @@ class MediaPlayer extends React.Component {
     );
 
     const progressBar = <ProgressBar
-      max={ this.videoRef.current !== null ? parseInt(this.videoRef.current.duration).toString() : '100' }
-      value={ this.videoRef.current !== null ? parseInt(this.videoRef.current.currentTime) : 0 }
+      max={ this.props.videoRef.current !== null ? parseInt(this.props.videoRef.current.duration).toString() : '100' }
+      value={ this.props.videoRef.current !== null ? parseInt(this.props.videoRef.current.currentTime) : 0 }
       buttonClick={ this.handleProgressBarClick.bind(this) }
     />;
 
     return (
       <section className={ styles.topSection }>
         <div className={ styles.playerSection }>
-          { this.props.mediaUrl !== null ? player : null }
-          { this.props.mediaUrl !== null ? playerControlsSection : null }
+          { this.props.mediaUrl === null ? null: playerControlsSection }
         </div>
-        { this.props.mediaUrl !== null ? progressBar : null }
+        { this.props.mediaUrl === null ? null: progressBar }
```

Comment:
same here with the spacing - `null: progressbar` looks like an object 😛

Reply:
ok fixed with this linting rule 
https://eslint.org/docs/rules/space-infix-ops